### PR TITLE
Eri9600/raster function service

### PR DESCRIPTION
**Pull request by eri9600**
@TADraeseke can you review when you have time. Thanks! 
As shown here: https://devtopia.esri.com/runtime/common-samples/issues/139

### CODE REVIEW

File: java/raster-function-service/src/main/java/com/esri/arcgisruntime/sample/rasterfunctionservice/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    // set the map to be displayed in this view
+    mMapView.setMap(map);
+  }
+
+  private void applyRasterFunction(Raster raster) {
+    // create raster function from json string
+    RasterFunction rasterFunction = RasterFunction.fromJson(getString(R.string.hillshade_simplified));
+    // get parameter name value pairs used by hillside
+    RasterFunctionArguments rasterFunctionArguments = rasterFunction.getArguments();
+    // get a list of raster names associated with a raster function
```
Comment (TADraeseke) [inline java/raster-function-service/src/main/java/com/esri/arcgisruntime/sample/rasterfunctionservice/MainActivity.java]: // get a list of raster names associated with the raster function

File: java/raster-function-service/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Raster Function Service
+Raster functions are operations performed on a raster to apply on-the-fly processing. This sample demonstrates how to create an image service raster and apply a hillshade raster function to it.
+
+![Raster Function Service App](raster-function-service.png)
+
+## How to use the sample
+Click on the button to apply the Raster service
```
Comment (TADraeseke) [inline java/raster-function-service/README.md]: . at the end

File: java/raster-function-service/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## How it works
+* Create `ImageServiceRaster` using a `URL` and load it.
+* Create `RasterFunction` using a `JSON` string.
+* Get raster function's arguments with `RasterFunctionArguments`
```
Comment (TADraeseke) [inline java/raster-function-service/README.md]: . at the end

**Comment by eri9600** [comment]
@TADraeseke Fixed the comments you made. Let me know if you find anything else. Thanks!

**Comment by TADraeseke** [comment]
Thanks @eri9600. Just a code -> format needed and you're good to merge!

**Comment by eri9600** [comment]
@TADraeseke Thanks for the review. Merging now.

**Pull request merged**